object_detection/detect.py

```diff
@@ -11,20 +11,41 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
-"""Main script to run pose classification and pose estimation."""
+"""Main script to run the object detection routine."""
 import argparse
 import sys
 import time
-
+from typing import List, NamedTuple
 import cv2
+
+from tflite_support.task import vision
+from tflite_support.task import core
+from tflite_support.task import processor
+
 import utils
 
-from object_detector import ObjectDetector
-from object_detector import ObjectDetectorOptions
+class Rect(NamedTuple):
```

Comment:
You shouldn't need to redefine these types: Rect, Category, Detection. There are equivalent data types in the TFLite Task Library. Instead, you should take the DetectionResult from the Task Library and update the utils.py file to receive a DetectionResult instance from the Task Library and show it, instead of List[Detection] like it's now.